### PR TITLE
testing->stable

### PR DESCRIPTION
**Pull request by ericLemanissier**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -13,9 +13,7 @@
     for settings, options, env_vars, build_requires, reference in builder.items:
         installers = ["nasm_installer/2.13.02@bincrafters/stable"]
         if build_shared.get_os() == "Windows":
-            installers.append("cygwin_installer/2.9.0@bincrafters/stable")
-            if os.getenv('MINGW_CONFIGURATIONS', ''):
-                installers.append("mingw_installer/1.0@conan/stable")
+            installers.append("msys2/20161025")
```
Comment (Croydon) [inline build.py]: This looks completely unnecessary as msys2 is already as a build requirement in the conanfile

File: build.py [hunk 2]
```diff
@@ -13,9 +13,7 @@
     for settings, options, env_vars, build_requires, reference in builder.items:
         installers = ["nasm_installer/2.13.02@bincrafters/stable"]
```
Comment (Croydon) [inline build.py]: nasm should be moved to the conanfile as well

File: conanfile.py [hunk 3]
```diff
@@ -48,9 +46,16 @@ def source(self):
         extracted_dir = 'x264-snapshot-%s-2245' % self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    @property
```
Comment (Croydon) [inline conanfile.py:49]: What is the purpose of this?

File: conanfile.py [hunk 4]
```diff
@@ -89,9 +105,12 @@ def package_info(self):
...
-        elif self._is_mingw_windows:
+        elif self._is_mingw:
             self.cpp_info.libs = ['x264.dll' if self.options.shared else 'x264']
         else:
             self.cpp_info.libs = ['x264']
         if self.settings.os == "Linux":
             self.cpp_info.libs.extend(['dl', 'pthread', 'm'])
+        if self.settings.os == "Android":
+            self.cpp_info.libs.extend(['dl', 'm'])
+        self.cpp_info.names['pkg_config'] = 'x264.pc'
```
Comment (Croydon) [inline conanfile.py]:
```suggestion
        self.cpp_info.names['pkg_config'] = 'x264'
```

Without `.pc` I assume

File: conanfile.py (hunk 3 above)
Comment (ericLemanissier) [reply conanfile.py:49]: I don't know. It was commited in https://github.com/bincrafters/conan-libx264/pull/8/commits/363c42a2c450054a790a3f5cee9c532127126642 and seems to use an undocumented feature of conan: https://github.com/conan-io/conan/blob/9e348424d85fa22b41c93cec4b0a113f25fef24d/conans/model/conan_file.py#L159
@akemimadoka can you explain us ?

File: conanfile.py (hunk 4 above)
Comment (ericLemanissier) [reply conanfile.py]: oops, wrong copy/paste.Thanks !

File: conanfile.py (hunk 3 above)
Comment (Croydon) [reply conanfile.py:49]: Despite no answer we might go ahead as @SSE4 approved that in #8 once...

**Review by Croydon — APPROVED** [review]

**Pull request merged**